### PR TITLE
sci-electronics/electric: Version bump to 9.07

### PR DESCRIPTION
**Pull request by flint2**
Signed-off-by: Guillermo Joandet <gjoandet@gmail.com>

### CODE REVIEW

File: sci-electronics/electric/electric-9.07.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+inherit desktop java-pkg-2
+
+DESCRIPTION="Complete Electronic Design Automation system for many forms of circuit design"
+HOMEPAGE="https://www.gnu.org/software/electric/electric.html"
+SRC_URI="mirror://gnu/electric/${PN}Binary-${PV}.jar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```
Comment (AndrewAmmerlaan) [inline sci-electronics/electric/electric-9.07.ebuild:15]: Empty vars can be dropped

File: sci-electronics/electric/electric-9.07.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+RDEPEND="
+	>=virtual/jre-1.6
+	sci-electronics/electronics-menu"
+DEPEND="app-arch/unzip"
```
Comment (AndrewAmmerlaan) [inline sci-electronics/electric/electric-9.07.ebuild:20]: Are you sure this is a DEPEND? unzip makes more sense as a RDEPEND or BDEPEND depending on whether it is executed during build- and/or runtime.

File: sci-electronics/electric/electric-9.07.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+S=${WORKDIR}
```
Comment (AndrewAmmerlaan) [inline sci-electronics/electric/electric-9.07.ebuild:22]: `WORKDIR` should be quoted.

File: sci-electronics/electric/electric-9.07.ebuild (hunk 1 above)
Comment (flint2) [reply sci-electronics/electric/electric-9.07.ebuild]: Solved in https://github.com/gentoo/gentoo/commit/7164e7eebc759abff6778213160c6513dddf98fd

File: sci-electronics/electric/electric-9.07.ebuild (hunk 2 above)
Comment (flint2) [reply sci-electronics/electric/electric-9.07.ebuild]: Solved in https://github.com/gentoo/gentoo/commit/7164e7eebc759abff6778213160c6513dddf98fd

File: sci-electronics/electric/electric-9.07.ebuild (hunk 3 above)
Comment (flint2) [reply sci-electronics/electric/electric-9.07.ebuild]: Solved in https://github.com/gentoo/gentoo/commit/7164e7eebc759abff6778213160c6513dddf98fd

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**